requests/check_pull.py

```diff
@@ -1,12 +1,13 @@
 import requests
```

Comment:
Pull Request prefix must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']